recruiter/forms.py

```diff
@@ -33,3 +34,8 @@ class UpdateRecruiterAccountSettingsForm(forms.ModelForm):
     class Meta:
         model = Recruiter
         fields = ["name", "company", "email", "phone_number"]
+
+    # Override the init method to add classes to the form fields
```

Comment:
you'd still need to write this kind of boilerplate code for all form object in your system - a batter approach would be to subclass ModelForm, add your customization and then make all your form classes inherit from that.

You can use multiple inheritance in Python, so this could probably be used just fine for classes that do not inherit directly from ModelForm as well.

Reply:
Let me just make sure I follow,
So I should create a class that inherits from ModelForm, overwrites the `init` method, and then make all the forms inherit from that class?

Reply:
@ifireball Ok I think I managed to do it